etc/checkfmt.sh

```diff
@@ -36,4 +36,14 @@ if [[ -n "$FMT_OUT" ]]; then
     HAS_FMT_ERR=1
 fi
 
+cd ..
+export PATH=$PATH:.
+FMT_OUT="$(flux fmt -c stdlib 2> /dev/null;)"
+if [[ -n "$FMT_OUT" ]]; then
```

Comment:
Acknowledging that the other checks are done this way, could `flux fmt -c ...` just return a non-zero status code on error, and we check the status code?

Reply:
Agreed a status code would be best.

Reply:
Thanks for the suggestion. 
Yes, I can do the changes to return non-zero status code for flux-fmt CLI. but the problem is it will require some changes in the `main.go`  and `root.go` for capturing the error return value. As this controls other CLI commands too so changing that for this task would not be ideal. (we can discuss)

Also, rustfmt --check returns file diff (if there are any) and similarly gofmt -d or -l returns file diff/ file list. So we are still maintaining that in flux-fmt where we are showing a message to the user.

On the other note, I realize there's no point returning an error with fmt -c as we are just checking if there's any flux file that is not fluxfmt-ed. So I have disabled our sub-command to return the error and just show the message returned in the root command.